scripts/zkai_attention_kv_sixteen_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,552 @@
+    add("unknown_field_injection", lambda _r, e, _s: e.__setitem__("sidecar_proof", []))
+    return cases
+
+
+def validate_result(result: dict[str, Any], envelope: dict[str, Any], source_input: dict[str, Any]) -> None:
+    mutation_results = result.get("mutation_results")
+    expected = build_result(envelope, source_input, mutation_results if isinstance(mutation_results, list) else [])
+    for key, value in expected.items():
+        if key == "mutation_results":
+            continue
+        if result.get(key) != value:
+            raise AttentionKvSixteenHeadFusedSoftmaxTableGateError(f"result drift for {key}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)